tests/run/fused_cpdef.pyx

```diff
@@ -50,36 +46,45 @@ def test_fused_cpdef(printto=None):
     <BLANKLINE>
     B, x is long 2 long
     """
-    if printto is None:
-        printto = sys.stdout
-    func1[int](None, 2, printto)
-    func1[long](None, 2, printto)
-    func1(None, 2, printto=printto)
+    func1[int](None, 2)
+    func1[long](None, 2)
+    func1(None, 2)
 
-    print >> printto
+    print
 
-    pyfunc[cy.int](None, 2, printto=printto)
-    pyfunc(None, 2, printto=printto)
+    pyfunc[cy.int](None, 2)
+    pyfunc(None, 2)
 
-    print >> printto
+    print
 
-    A.meth[cy.int](A(), 2, printto=printto)
-    A.meth(A(), 2, printto=printto)
-    A().meth[cy.long](2, printto=printto)
-    A().meth(2, printto=printto)
+    A.meth[cy.int](A(), 2)
+    A.meth(A(), 2)
+    A().meth[cy.long](2)
+    A().meth(2)
 
-    print >> printto
+    print
 
-    B().meth(2, printto=printto)
+    B().meth(2)
 
 
 midimport_run = io.StringIO()
-midimport_run.write = lambda c: io.StringIO.write(midimport_run, unicode(c))
+try:
+    unicode
+except NameError:
```

Comment:
I don't think this works. Cython should be renaming `unicode` to `str` in Py3 here, which means that it'll always end up in the `else:` clause.

Reply:
Ah, I was wondering why it seemed to be working even before I added the `unicode`.